include/crow/utility.h

```diff
@@ -503,7 +504,7 @@ template <typename F, typename Set>
             using arg = typename std::tuple_element<i, std::tuple<Args...>>::type;
         };
 
-        inline static std::string base64encode(const char* data, size_t size, const char* key = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/")
+        inline static std::string base64encode(const unsigned char* data, size_t size, const char* key = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/")
```

Comment:
Probably `uint8_t` would be more descriptive for this parameter ;)

Reply:
I don't believe so, I've never seen `uint8_t*` be used to refer to a string of characters. `unsigned char*` shows that "this is a string, but it uses unsigned characters instead of normal ones"